### PR TITLE
Add df.to_eto functionality

### PR DESCRIPTION
**Pull request by changhiskhan**

### CODE REVIEW

File: eto/fluent/datasets.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
+import os
```
Comment (eddyxu) [inline eto/fluent/datasets.py:1]: duplicated import

File: eto/fluent/client.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    api = cls_map[api_name]
+    return api(client)
+
+
+def configure(
+    account: Optional[str] = None,
+    token: Optional[str] = None,
+    use_ssl: bool = True,
+    port: Optional[int] = None,
+    tmp_workspace_path: Optional[str] = None,
```
Comment (eddyxu) [inline eto/fluent/client.py:55]: should we configure tmp_workspace_path within the service itself?

File: eto/fluent/datasets.py [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+    uri = _normalize_uri(get_dataset(dataset_name).uri)
+    if isinstance(columns, str):
+        columns = [columns]
+    dataset = RikaiDataset(uri, columns)
+    if limit is None or limit <= 0:
+        return pd.DataFrame(dataset)
+    else:
+        rows = [None] * limit
+        i = 0
+        for i, r in enumerate(dataset):
```
Comment (eddyxu) [inline eto/fluent/datasets.py:66]: can we use islice?

File: eto/fluent/datasets.py [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+    partition: Optional[str] or list[str], default None
+        Which columns to partition by
+    mode: str, default 'append'
+        Controls behavior if the dataset_name already exists
+    async_: bool, default False
+        If True then return immediately with the ingestion Job
+    max_wait_sec: int, default -1
+        Maximum number of seconds to wait. If negative then wait forever.
+        If async_ is True then this is ignored
+    infer_schema: bool, default True
```
Comment (eddyxu) [inline eto/fluent/datasets.py:101]: shall we only need one of `infer_schema` or `schema`?
As `infer_schema === (schema is None)`?

File: eto/util.py [hunk 5]
```diff
@@ -43,3 +44,17 @@ def get_dataset_ref_parts(qualified_name: str):
     else:
         project_id, dataset_id = None, qualified_name
     return project_id, dataset_id
+
+
+def add_method(cls):
```
Comment (eddyxu) [inline eto/util.py:49]: `patch_method`?
This decorator is nice tho

**Review by eddyxu — APPROVED** [review]
Small nits, but overall lgtm

File: eto/fluent/datasets.py (hunk 1 above)
Comment (changhiskhan) [reply eto/fluent/datasets.py:1]: fixed

File: eto/fluent/client.py (hunk 2 above)
Comment (changhiskhan) [reply eto/fluent/client.py:55]: Once we have a UI we can configure this in the service. Then the SDK can pull from the service automatically but let the user override it if they need.

File: eto/util.py (hunk 5 above)
Comment (changhiskhan) [reply eto/util.py:49]: here we're patching the class by adding a method. `patch_method` sounds like we're patching the method rather than the class.

**Pull request merged**